d5005/hardware/ofs_d5005/build/scripts/run.sh

```diff
@@ -34,21 +34,15 @@ if [ ${BSP_FLOW} = "afu_flat_kclk" ]; then
 fi
 
 cd "$SCRIPT_DIR_PATH/.." || exit
+#export PYTHONPATH="$OFS_ASP_ROOT"/build/opae/install/lib/python3*/site-packages
 
 if [ -n "$PACKAGER_BIN" ]; then
   echo "Selected explicitly configured PACKAGER_BIN=\"$PACKAGER_BIN\""
 elif [ -z "$OFS_ASP_ENV_USE_BSP_PACKAGER" ] && PACKAGER_BIN="$(command -v packager)"; then
   echo "Detected PACKAGER_BIN=\"$PACKAGER_BIN\" from \$PATH search"
 else
-  echo "Attempting fallback to BSP copy of packager"
-  if [ -f ./tools/packager ]; then
-    chmod +x ./tools/packager
-    PACKAGER_BIN=$(readlink -f ./tools/packager)
```

Comment:
Are you sure you want to remove these `PACKAGER_BIN` assignments? By my reading, the only way line 45 (to update `PYTHONPATH` will ever execute is if `PACKAGER_BIN` is both unset/empty at the start of the script (thus skipping the line 40 path) and no `packager` was found in the `$PATH` search on line 41's predicate.

If this last conditional path is ever hit, `$PACKAGER_BIN` is an empty string and line 48 will immediately fail when it tries to execute it.

Reply:
The last conditional could also be hit if `$OFS_ASP_ENV_USE_BSP_PACKAGER` is nonempty, but `$PACKAGER_BIN` will still be empty in this case too, right?

Reply:
yes i tried out oneapi .fpga compilation and it works, if we keep the if [ -f ./tools/packager ]; somehow its not copied to /tmp workspace of oneapi compilation and it cant find it and it fails

Reply:
yes $OFS_ASP_ENV_USE_BSP_PACKAGER is non empty , $PACKAGER_BIN is empty for our purposes when we are running this script i think

Reply:
> $PACKAGER_BIN is empty for our purposes when we are running this script i think

But if `$PACKAGER_BIN` is empty when line 48 executes, the script will fail and exit immediately. If this version is working for you, that means `$PACKAGER_BIN` is non-empty; it must be getting set somewhere.

Either it's being set/exported before the script starts, or it's being assigned by the `$PATH` search with `PACKAGER_BIN="$(command -v packager)"`. Neither of those cases result in the last branch of the conditional running.

Reply:
what do you suggest?

Reply:
Personally, I don't care what you do with those lines. I expect this path is not executed by the end customer because they'll be using `packager` directly from `$PATH` search as a result of having installed the OPAE tools package somehow.

But putting my PR Reviewer hat on, just by visual inspection I don't see how those lines ever execute in a way that doesn't immediately exit a few lines later. `$PACKAGER_BIN` is empty if you aren't assigning to it. Have you tried mentally stepping through the code to understand my logic?

Reply:
Yea i guess you are right, I am trying to set PACKAGER_BIN but it doesnt work, it errors out when i try actual oneapi compile run

Reply:
@chipb-intel  than you for feedback. The issue is maybe you mentioned earlier it's not working well with 3*, when i use pythonpath explicitly as 3.8 it works fine. I made that update. If you think there is better way to do that let me know